data/annotations/transform_polygon_annotations_to_pixels.ipynb

```diff
@@ -89,23 +100,94 @@
+        "    dfs = []\n",
+        "    for tif_file in glob.glob(os.path.join(images_folder, regex)):\n",
+        "            tif_file = tif_file.replace(\"\\\\\",\"/\")\n",
+        "            print(tif_file)\n",
+        "            name_tif_file = tif_file.split(\"/\")[-1].split(\".\")[0]\n",
+        "            with rasterio.open(tif_file) as dataset:\n",
+        "                dfs += [\n",
+        "                extract_dataframe_pixels_values_from_tif_and_polygons(\n",
+        "                            tif_dataset=dataset,\n",
+        "                            polygon_gdf=annotations_polygons_gdf[\n",
+        "                                annotations_polygons_gdf[\"name\"] == \"Annotations_Coepelduynen_2023\"\n",
+        "                            ],\n",
+        "                            name_tif_file=tif_file.split(\"/\")[-1],\n",
+        "                            name_annotations=\"Annotations_Coepelduynen_2023\",\n",
+        "                        )          \n",
+        "                ]\n",
+        "            \n",
+        "    df = pd.concat(dfs)\n",
+        "\n",
+        "    # Annotations for specific satellite images\n",
         "    dfs = []\n",
         "    for tif_file in glob.glob(os.path.join(images_folder, regex)):\n",
         "        tif_file = tif_file.replace(\"\\\\\",\"/\")\n",
         "        print(tif_file)\n",
-        "        name_tif_file = tif_file.split(\"/\")[-1].split(\".\")[0]\n",
+        "        name_tif_file = tif_file.split(\"/\")[-1].split(\".\")[0].split(\"_\")[0]+\"_annotations\"\n",
+        "        print(name_tif_file)\n",
         "        with rasterio.open(tif_file) as dataset:\n",
-        "            dfs += [\n",
+        "                dfs += [\n",
         "                extract_dataframe_pixels_values_from_tif_and_polygons(\n",
-        "                    tif_dataset=dataset, polygon_gdf=annotations_polygons_gdf,name_tif_file=name_tif_file\n",
-        "                )\n",
-        "            ]\n",
+        "                            tif_dataset=dataset,\n",
+        "                            polygon_gdf=annotations_polygons_gdf[\n",
+        "                                annotations_polygons_gdf[\"name\"] == name_tif_file\n",
+        "                            ],\n",
+        "                            name_tif_file=tif_file.split(\"/\")[-1],\n",
+        "                            name_annotations=name_tif_file,\n",
+        "                        )          \n",
+        "                ]\n",
         "        \n",
-        "    df = pd.concat(dfs)\n",
-        "    df.to_parquet(pixel_filepath)"
+        "    \n",
+        "    dfs = pd.concat(dfs)\n",
+        "    df = pd.concat([dfs,df])\n",
+        "    \n",
+        "    # Extra check to see if all the correct bands are found\n",
+        "    #if \"ndvi\" not in df.columns:\n",
+        "\n",
+        "    #    if \"n\" in df.columns:\n",
```

Comment:
I would remove these commented lines, just to keep it clean (and like we discussed earlier, let's keep this logic in the extractor).